packages/components/src/components/popup/popup.scss

```diff
@@ -0,0 +1,151 @@
+.dc-popup {
+    &-header {
+        padding: 0;
+
+        &__close-icon {
+            position: absolute;
+            right: 2%;
+
+            @include mobile {
+                right: 4%;
+            }
+        }
+
+        &__title {
+            @include mobile {
+                display: flex;
+                flex-direction: row;
+                align-items: center;
+            }
+        }
+
+        &__content {
+            display: flex;
+            align-items: center;
+        }
+
+        &__icon {
+            position: absolute;
+            right: 4%;
+
+            @include mobile {
+                right: 10%;
+            }
+        }
+
+        &__button {
+            margin: 0 1.6em !important;
+            width: 13rem !important;
+
+            @include mobile {
+                margin: 0 2.6em !important;
+                width: 3.2rem !important;
+            }
+        }
+
+        &__banner {
+            background-color: var(--status-warning);
+            border-radius: 100px;
+            padding: 0.4rem 0.8rem;
+            height: 2.2rem;
+            display: flex;
+            align-items: center;
+
+            @include mobile {
+                background-color: var(--general-main-1);
+                margin-right: 1.6rem;
+                margin-left: 1.6rem;
+            }
+        }
+    }
+    &-body {
+        padding: 0 !important;
+    }
+
+    &-footer {
+        display: flex;
+        justify-content: flex-end;
+        padding: 1.6rem;
+        position: fixed;
+        width: 100%;
+        bottom: 0;
+
+        &__separator {
+            border-top: 2px solid var(--general-section-1);
+        }
+    }
+}
+
+.dc-modal {
+    &-header {
+        &--popup {
+            height: 12.4rem;
+            align-items: flex-start;
+
+            @include mobile {
+                height: 7.8rem;
+            }
+        }
+        &__title {
+            &--popup {
+                @include mobile {
+                    align-items: flex-start;
+                }
+            }
+        }
+    }
+}
+
+.dc-tabs {
```

Comment:
this needs to be scoped properly, if not it will override the styles in `dc-tabs` once the component is loaded with this `sass` import

Reply:
In this case it should be fine as there are no rules added for `.dc-tabs` itself, it's definitely not ideal but the the prefixing that happens in the tabs component isn't either (it prefixes the custom classNames with `.dc-tabs` everywhere).